pkg/console/controllers/advancedclustermanagement/managedclusters/controller.go

```diff
@@ -90,12 +90,15 @@ func NewManagedClusterController(
 	}
 
 	configV1Informers := configInformer.Config().V1()
+	managedClusterViewInformer := dynamicInformers.ForResource(api.ManagedClusterViewGroupVersionResource)
 
 	return factory.New().
 		WithFilteredEventsInformers( // configs
 			util.IncludeNamesFilter(api.ConfigResourceName),
 			configV1Informers.Consoles().Informer(),
 			operatorConfigInformer.Informer(),
+			managedClusterInformers.Informer(),
+			managedClusterViewInformer.Informer(),
```

Comment:
I don't think this will work as expected. The name filter on L93 wouldn't match any resources. We probably need a new `WithFilteredEventsInformers` call. I think I used a label filter in the past to make sure we are only watching the resources managed by console.